gradle.properties

```diff
@@ -6,4 +6,4 @@ deployUrl = https://oss.sonatype.org/service/local/staging/deploy/maven2/
 # Enable the daemon by adding org.gradle.daemon in USER_HOME/.gradle/gradle.properties
 org.gradle.parallel=true
 
-java6Home=/Library/Java/JavaVirtualMachines/1.6.0_65-b14-462.jdk/Contents/Home
+java6Home=/Library/Java/JavaVirtualMachines/1.6.0.jdk/Contents/Home
```

Comment:
ok, but what's the difference then with other properties in this file ? To me, if a file contains some local things, it should be ignored from git. Because otherwise, people will keep having untracked modifications locally and it's quite a burden.  Couldn't we get rid of this file then and put in the global README it's content, saying that devs needs to have this file with this content, modified as they need ?